backend/functions/lib/adapters/types/cases.d.ts

```diff
@@ -41,5 +48,15 @@ export interface DxtrTransactionRecord {
 }
 
 export interface DxtrPartyRecord {
-  partyName: string;
+  name: string;
+  address1?: string;
+  address2?: string;
+  address3?: string;
+  address4?: string;
+}
+
```

Comment:
This is a duplicate of Debtor interface.  Let's nix one.  And maybe it should be called something like Party, since debtor is only one type of party.